RestartManager.js

```diff
@@ -2,6 +2,7 @@ const log = require("./logging");
 const NativeCodePush = require("react-native").NativeModules.CodePush;
 
 const RestartManager = (() => {
+    let _in_progress = false;
```

Comment:
I'd prefer to call this _restartInProgress, but either way is fine. But, we should use camel casing for the name to be consistent with others, and sort it alphabetically.